hive-hooks/path-conversion-hook/pom.xml

```diff
@@ -0,0 +1,96 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.expediagroup.apiary</groupId>
+        <artifactId>hive-hooks-parent</artifactId>
+        <version>7.2.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>path-conversion-hook</artifactId>
+    <name>Path Conversion Hive Hook</name>
+    <description>Hive Hook to convert matching path regex to another</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <lombok.version>1.18.16</lombok.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombok.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-metastore</artifactId>
+            <version>${hive.version}</version>
+            <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.hbase</groupId>
+                    <artifactId>hbase-client</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.logging.log4j</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-exec </artifactId>
+            <version>${hive.version}</version>
+            <scope>provided</scope>
```

Comment:
Can you use the `<classifier>core</classifier>` version of this dependency? It contains fewer "surprises" which have caused much pain in the past.